sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/InternalRow.scala

```diff
@@ -53,70 +53,8 @@ abstract class InternalRow extends Row {
 
   // A default implementation to change the return type
   override def copy(): InternalRow = this
-  override def apply(i: Int): Any = get(i)
```

Comment:
The origin code is weird. We have defined `get(i: Int) = apply(i)`, and we should implement `apply` in subclasses. However, we change this definition here and subclasses of `InternalRow` have to implement `get`.